src/core/tests/playwright/conftest.py

```diff
@@ -0,0 +1,69 @@
+import subprocess
+import pytest
+import requests
+
+headers = {
+    "Authorization": "Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJmcmVzaCI6ZmFsc2UsImlhdCI6MTcxNDY3NTMzNywianRpIjoiZTA3NjRiZmQtMDUzOC00NDg4LTljZmMtY2NkZmFjZGM2MGU2IiwidHlwZSI6ImFjY2VzcyIsInN1YiI6ImFkbWluIiwibmJmIjoxNzE0Njc1MzM3LCJjc3JmIjoiZWRmOTc2NTktNzAxMy00NzViLTg3ZDUtNTM0NTczM2VmY2NlIiwidXNlcl9jbGFpbXMiOnsiaWQiOjEsIm5hbWUiOiJBcnRodXIgRGVudCIsInJvbGVzIjpbMV19fQ.siZID18vdHC1tyQnW8B9KuwqU-B2QJZHPBpknAyB-qg"
+}
+
+
+@pytest.fixture(scope="session")
+def temporary_taranis_instance():
+    subprocess.run(["chmod", "+x", "./tests/playwright/temporary_taranis_instance.sh", "&>/dev/null &"])
+    result = subprocess.run(["./tests/playwright/temporary_taranis_instance.sh", "up", "&>/dev/null &"])
+    if result.returncode != 0:
+        pytest.fail("Script failed")
+    yield result.stdout
+    subprocess.run(["./tests/playwright/temporary_taranis_instance.sh", "down"])
+
```

Comment:
Could be replaced by a simple call to `npm run build` and directly served by via python, so only a single process needs to be running.

Reply:
possibly resolved